wgsl/index.bs

```diff
@@ -8547,8 +8547,7 @@ See [[#builtin-inputs-outputs]] for how to declare a built-in value.
       <td>fragment
       <td>input
       <td>vec4&lt;f32&gt;
-      <td style="width:50%">Framebuffer position of the current fragment, using normalized homogeneous
-      coordinates.
+      <td style="width:50%">Framebuffer position of the current fragment in framebuffer space.
```

Comment:
"Frambuffer coordinates" are explained in the Rasterization section of WebGPU. https://gpuweb.github.io/gpuweb/#rasterization

I'm ok to land this and update it and make an editorial adjustment to say

> Position of the current current fragment, in framebuffer coordinates.

Reply:
Thank you! I turned “framebuffer space” into a link to the section you provided :)